torch/fx/graph.py

```diff
@@ -2203,6 +2191,11 @@ def on_generate_code_context_manager():
 
         return on_generate_code_context_manager()
 
+    def _clear_nodes(self) -> None:
+        for node in reversed(self.nodes):
```

Comment:
Do we have some notion of how this may affect runtime? I.e in case a subtracer fails, but we continue dynamo graph capture, how long can we expect erasing all the node's in the subtracer to be?